.eslintrc

```diff
@@ -4,5 +4,55 @@
     "env": {
         "node": true,
         "jest": true
+    },
+    "rules": {
+        "import/order": [
```

Comment:
Don't you want to add it here https://github.com/gravity-ui/eslint-config ? If someone wants to set the same rules for themselves, then they just import them into their project